styles/style.css

```diff
@@ -0,0 +1,171 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+@import url('https://fonts.googleapis.com/css?family=Noto+Serif+TC');
+
+
+* {
+    font-family: 'Roboto', sans-serif;
+}
+
+.container {
+    padding: 2rem;
+}
+
+.header {
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.header > li:nth-child(2) {
+    margin-left: auto;
+}
+
+.header > li:nth-child(3) {
+    margin-left: 10rem;
+    margin-right: 5rem;
+}
+
+ul {
+    list-style-type: none;
+  }
+
+.corporate {
+    display: flex;
+    height: 35vh;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+    background-color: #0B0C10;
+    color: #66FCF1;
+}
+
+.container sub {
+    position: static; 
+    line-height: normal;
+    color: #C5C6C7;
+}
+
+.corporate a {
+    border-color: gray;
+    border-radius: 10%;
+    border-style: solid;
+    padding: .4rem;
+    text-decoration: none;
+    color: white;
+}
+
+.corporate a:hover {
+    color: yellow;
+}
+
+.quote_1, .quote_2 {
+    display: flex;
+    height: 25vh;
+    justify-content: center;
+    flex-direction: column;
+    flex-wrap: wrap;
+    font-family: 'Noto Serif TC', serif;
+    font-style: italic;
+    border-top-style: solid;
+    border-bottom-style: solid;
+    border-top-color: gray;
+    border-bottom-color: gray;
+    border-width: thin;
+    background-color: #C5C6C7;
+}
+
+.quote_1 blockquote, .quote_2 blockquote {
+    align-items: center;
+}
+
+.quote_1 p, .quote_2 p {
```

Comment:
did you mean to keep p in background color even when page is resized? i think wrapping the p in div and setting margin could help with that?